HW2.Rmd

```diff
@@ -0,0 +1,110 @@
+---
+title: "HW2"
+author: "Roxanne Felig"
+date: "2/7/2021"
+output: html_document
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Assignment 02: Explore gapminder and use dplyr
+
+```{r load packages}
+# load your packages here:
+library(gapminder)
+library(tidyverse)
+library(dplyr)
+```
+
+### 1.1 Use filter() to subset the gapminder data to three countries of your choice in the 1970’s.
+```{r filter}
+gapminder %>%
+  filter(country %in% c("Argentina","Chile","Germany") & year>1969 & year<1980)
```

Comment:
Oh, this is some handy code. I did mine a bit differently and arguably a bit wordier. I'll be sure to use this method, thanks!